pyproject.toml

```
@@ -11,7 +11,7 @@ description = "Tooling for building derived datasets in BigQuery"
 readme = "README.md"
 requires-python = ">=3.11"
 dynamic = ["dependencies"]
-version = "2024.5.1"
+version = "2025.1.15"
```

@scholtzan should I revert this value back?

---

that's fine, although it should be 2025.1.1

```suggestion
version = "2025.1.1"
```